package.json

```diff
@@ -138,5 +138,9 @@
     "tslint-react-hooks": "^2.2.1",
     "typescript": "3.8.2",
     "webpack": "4.46.0"
+  },
+  "engines": {
+    "node": ">=14.17.5",
```

Comment:
think we'll want to make sure we're using the same thing as the dockerfile https://github.com/dcos-labs/ui-kit/blob/master/Dockerfile#L1